modal/image.py

```diff
@@ -882,6 +882,19 @@ def from_dockerhub(
         setup_dockerfile_commands: List[str] = [],
         force_build: bool = False,
         **kwargs,
+    ) -> "_Image":
+        deprecation_warning(
+            date(2023, 8, 25), "`Image.from_dockerhub` is deprecated. Use `Image.from_registry` instead."
+        )
+        return _Image.from_registry(tag, setup_dockerfile_commands, force_build, **kwargs)
```

Comment:
super minor nit (not blocking) but we generally use "newspaper style" function ordering in Modal, i.e. callee is defined before caller

Reply:
Got it!